.circleci/config.yml

```diff
@@ -79,53 +79,65 @@ jobs:
                   name: Update branch protection rule from test configuration
                   command: ./scripts/test.py required-status >required_status && cat required_status && ./scripts/update_branch_rule.py required_status
 
-    #build-macos:
-    #    macos:
-    #        xcode: "10.1.0"
-    #    working_directory: /Users/distiller/coda
-    #    environment:
-    #        HOMEBREW_LOGS: /Users/distiller/homebrew.log
-    #        OPAMYES: 1
-    #    steps:
-    #        - run:
-    #            name: Make /nix paths
-    #            command: |
-    #                sudo mkdir /nix
-    #                sudo chown distiller /nix
-    #        - checkout
-    #        - restore_cache:
-    #              keys:
-    #                  - homebrew-v1-{{ checksum "scripts/macos-setup.sh" }}
-    #                  - homebrew-v1-
-    #        - restore_cache:
-    #              keys:
-    #                  - opam-v3-{{ checksum "src/opam.export" }}
-    #                  - opam-v3-
-    #        - run: git submodule sync && git submodule update --init --recursive
-    #        - run:
-    #              name: Download Deps -- make macos-setup-download
-    #              command: ./scripts/skip_if_only_frontend.sh make macos-setup-download
-    #        - run:
-    #              name: Compile Deps -- make macos-setup-compile
-    #              command: ./scripts/skip_if_only_frontend.sh make macos-setup-compile
-    #        - save_cache:
-    #              key: homebrew-v1-{{ checksum "scripts/macos-setup.sh" }}
-    #              paths:
-    #                  - "/usr/local/Homebrew"
-    #                  - "/Users/distiller/Library/Caches/Homebrew"
-    #        - save_cache:
-    #              key: opam-v3-{{ checksum "src/opam.export" }}
-    #              paths:
-    #                  - "/Users/distiller/.opam"
-    #        - run:
-    #              name: Build OCaml
-    #              command: ./scripts/skip_if_only_frontend.sh bash -c 'eval `opam config env` && make build 2>&1 | tee /tmp/buildocaml.log'
-    #        - run:
-    #              name: Record Constraint System Digests
-    #              command: ./scripts/skip_if_only_frontend.sh bash -c 'src/_build/default/app/cli/src/coda.exe client constraint-system-digests | tee /tmp/constraint-system-digests.log'
-    #        - run:
-    #              name: Run all tests (on master)
-    #              command: echo "FIXME Tests not yet working on mac"
+    build-macos:
+       macos:
+           xcode: "10.1.0"
```

Comment:
I'm not sure if this is the right version we should be building for, but I guess we should do it for now.